client/components/domains/contact-details-form-fields/managed-contact-details-form-fields.jsx

```diff
@@ -151,7 +151,7 @@ export class ManagedContactDetailsFormFields extends Component {
 			isError: !! form[ camelName ]?.errors?.length,
 			errorMessage: customErrorMessage || getFirstError( form[ camelName ] ),
 			onChange: this.handleFieldChangeEvent,
-			onBlur: this.handleBlur,
+			onBlur: this.handleBlur( name ),
```

Comment:
Should this remain a function argument? 

```suggestion
			onBlur: () => this.handleBlur( name ),
```